src/app-layer-htp-libhtp.h

```diff
@@ -139,6 +199,15 @@
 #define htp_tx_request_header_index(tx, i)    htp_table_get_index(tx->request_headers, i, NULL);
 #define htp_headers_t                         htp_table_t
 
+// Functions introduced to handle opaque htp_tx_data_t:
+#define htp_tx_data_len(d)  d->len
+#define htp_tx_data_data(d) d->data
+#define htp_tx_data_tx(d)   d->tx
+
+// Functions introduced to handle opaque htp_conn_t:
+#define htp_conn_request_data_counter(c)  c->in_data_counter
```

Comment:
we need to use proper macro hygiene, so `(c)->in_data_counter`.

This is an issue with previous patches as well, pointed out during some `clang-tidy` experiments.

Reply:
:-p